.github/workflows/check-traceability.yml

```diff
@@ -0,0 +1,47 @@
+name: Check PR traceability
+
+on:
+  push:
+    branches: [master, "release/*"]
+  pull_request:
+    branches: [master, "release/*"]
```

Comment:
I'm not sure about validating this in CI. I don't think any minor changes should require an issue. And many external contributors open small PRs without linked issues for convenience.

Perhaps one option would be to allow setting "let's do it" on PRs to disable this check but then you wouldn't want to have to commit to re-trigger it. Would need to change the `on:` to something like message update

Reply:
This is fine. I will enable this only for FB and core contributors. I think it is important we force traceability for all PRs.

Reply:
How would you do that? the workflow runs on all PRs